lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingTransportClient.cs

```diff
@@ -21,36 +21,35 @@
 using System.Net;
```

Comment:
`Dispose(bool)` is only called with `true` as the value. Maybe we can dispose of the method altogether?